code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -54,12 +55,45 @@
 	icon_living = "combatbear"
 	icon_dead = "combatbear_dead"
 	faction = list("russian")
+	butcher_results = list(/obj/item/weapon/reagent_containers/food/snacks/meat/slab/bear = 5, /obj/item/clothing/head/bearpelt = 1, /obj/item/bear_armor = 1)
+	melee_damage_lower = 25
+	melee_damage_upper = 35
+	armour_penetration = 20
 	health = 120
 	maxHealth = 120
 
 /mob/living/simple_animal/hostile/bear/Process_Spacemove(movement_dir = 0)
 	return 1	//No drifting in space for space bears!
 
+/mob/living/simple_animal/hostile/bear/update_icons()
+	..()
+	if(armored)
+		var/image/B = "armor_bear"
```

Comment:
it works by coincidence, this is not how you do images!
(the ```overlays``` list can take a string (```"armor_bear"```) to make an overlay from if there is a magic state in their icon file with that name, this "image/B" gets passed through ```add_overlay()``` and eventually ends up in the bear's ```overlays```, however B is typed as an image, when it is a string.)

make this ```var/image/B = image(icon = 'fucking_bear_thingy.dmi', icon_state = "armor_bear")```